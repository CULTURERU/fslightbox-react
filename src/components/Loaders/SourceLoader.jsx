import React from 'react';

const SourceLoader = () =>
    (
        <div className="fslightbox-loader">
            <div></div>
            <div></div>
            <div></div>
            <div></div>
        </div>
    );

export default SourceLoader;