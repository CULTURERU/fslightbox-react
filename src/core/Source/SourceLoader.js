/**
 * @class
 */
export function SourceLoader(fsLightbox) {

}