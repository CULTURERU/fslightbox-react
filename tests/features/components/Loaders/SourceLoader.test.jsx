import React from 'react';
import { shallow } from "enzyme";
import SourceLoader from "../../../../src/components/Loaders/SourceLoader";

const sourceLoader = shallow(<SourceLoader/>);

describe('SourceLoader DOM', () => {
    it('should match snapshot', () => {
        expect(sourceLoader).toMatchSnapshot();
    });
});