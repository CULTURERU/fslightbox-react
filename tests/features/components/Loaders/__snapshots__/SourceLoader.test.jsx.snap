// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SourceLoader DOM should match snapshot 1`] = `
<div
  className="fslightbox-loader"
>
  <div />
  <div />
  <div />
  <div />
</div>
`;
